File: Daily Test/Java/Maximum Of Odd And Even Digit Count 16.4.2026.java
import java.util.*;

public class Hello {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int n = s.nextInt();
        int odd = 0, even = 0;

        while(n > 0) {
            if((n % 10) % 2 == 0)
                even++;
            else
                odd++;
            n /= 10;
        }

        System.out.println(Math.max(odd, even));
    }
}

File: Daily Test/C/Maximum Of Odd And Even Digit Count 16.4.2026.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int odd = 0, even = 0;

    while(n > 0) {
        if((n % 10) % 2 == 0)
            even++;
        else
            odd++;
        n /= 10;
    }

    printf("%d", (odd > even) ? odd : even);
    return 0;
}